src/Components/ModalCard/styles.ts

```diff
@@ -0,0 +1,77 @@
+import { Card, CardActionArea, styled as muiStyled } from "@mui/material";
```

Comment:
what is muiStyled?

Reply:
because i used styled-components too and them i called stayle.